testing/test_package/lib/example.dart

```diff
@@ -355,6 +355,37 @@ class Dog implements Cat, E {
   /// Don't define this:  {@macro ThatDoesNotExist}
   void withUndefinedMacro() {}
 
+  /// Animation method
```

Comment:
Another interesting case would be a doc comment that has an animation on the first line:

/// Hello animation world:  {@animation stuffAnimation 100 100 http://etc}

Basically, to exercise what happens when an animation might appear in a oneLineDoc (the summary of each method displayed on a Class page, for example).  It's probably desirable not to have the animation appear in the summary of each method, and I _think_ that's what may happen, but a test should help.

And other cases where @animation is inlined:

/// A test
/// This is something cool {@animation a 100 200 http://stuff} that I am inlining.

Reply:
OK, I've added tests for both inlined animations and oneLineDoc substitution.  It seems that the oneLineDoc doesn't incorporate the animations in the first place (probably because the animation code is preceded by a blank line), so no problem there, and the inline test found a problem (I was indenting the first line after the animation code, which made markdown think that it was code), which I fixed.